src/olympia/lib/settings_base.py

```diff
@@ -246,8 +246,8 @@ def get_db_config(environ_var, atomic_requests=True):
 # Static
 STATIC_ROOT = path('site-static')
 # Allow overriding static/media url path prefix
-STATIC_URL_PREFIX = env('STATIC_URL_PREFIX', default='/static-server/')
-MEDIA_URL_PREFIX = env('MEDIA_URL_PREFIX', default='/user-media/')
+STATIC_URL_PREFIX = env('STATIC_URL_PREFIX')
+MEDIA_URL_PREFIX = env('MEDIA_URL_PREFIX')
```

Comment:
what's the rationale behind specifying these defaults in the Dockerfile rather than here in the settings?

Reply:
The values need to be available to vite via the environment. Setting the default in the base image guarantees these values exist. If we reverted this and you ran `npm run build` without defining the values on the environment explicitly, zod would complain and the build would fail.